### PR TITLE
Added nether bricks block and its variants

### PR DESCRIPTION
**Pull request by mmm545**

### CODE REVIEW

File: server/block/nether_bricks.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import "github.com/df-mc/dragonfly/server/world"
+
+// NetherBricks are blocks used to form nether fortresses in the Nether.
+// Red Nether bricks, Cracked Nether bricks and Chiseled Nether bricks are decorative variants that do not naturally generate.
+type NetherBricks struct {
+	solid
+	bassDrum
+
+	Type NetherBricksType
```
Comment (Sandertv) [inline server/block/nether_bricks.go:11]: Don't forget to add documentation for this field.

**Review by Sandertv — COMMENTED** [review]
Just one small comment, looks good otherwise!

File: server/block/nether_bricks.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+import "github.com/df-mc/dragonfly/server/world"
+
+// NetherBricks are blocks used to form nether fortresses in the Nether.
+// Red Nether bricks, Cracked Nether bricks and Chiseled Nether bricks are decorative variants that do not naturally generate.
+type NetherBricks struct {
+	solid
+	bassDrum
+
+	// NetherBricksType is the type of nether bricks of the block
```
Comment (T14Raptor) [inline server/block/nether_bricks.go:11]: A period is needed here.

Comment (mmm545) [reply server/block/nether_bricks.go]: Done

**Review by Sandertv — APPROVED** [review]
Looks good, thanks for your PR!

**Pull request merged**